src/api/hooks/useGetDelegationNodeInfo.ts

```diff
@@ -36,11 +37,13 @@ export function useGetDelegationNodeInfo({
   const [delegatedStakeAmount, setDelegatedStakeAmount] = useState<string>();
   const [networkPercentage, setNetworkPercentage] = useState<string>();
   const [isQueryLoading, setIsQueryLoading] = useState<boolean>(true);
+  const [validatorStatus, setValidatorStatus] = useState<Types.MoveValue[]>();
 
   useMemo(() => {
     if (!isLoading) {
       const fetchData = async () => {
         setCommission(await getValidatorCommission(client, validatorAddress));
+        setValidatorStatus(await getValidatorState(client, validatorAddress));
```

Comment:
Does this function handle pending active and pending inactive states as well?

Reply:
i was hoping we don't overwhelm users too much with different validator states. if validator's inactive, we show them as inactive, the rest's shown as active. lmk if you don't think it's the best approach